README.md

```diff
@@ -324,13 +324,23 @@ which means they can't be joined up.
 
 ### Entity Table Check Job
 
-The entity table check job will run every night and sends data to verify that the latest version of an entity table in BigQuery matches the database. The job is defaulted to false but can be changed by updating the configuration option in
-`config/initializers/dfe_analytics.rb`:
+If you have *Sidekiq* and *Sidekiq-Cron* installed, you might want to enable the *Entity Table Check Job* to verify that the latest version of an entity table in Big Query matches the database. Ideally the job would be scheduled to run every night.
```

Comment:
Question here. Do we have to limit this to SideKiq ? I expect this would still work if they use Rescue or Delayed job for example. It's just we've passed on the responsibility of scheduling to the application.

Reply:
No we don't need to limit it, it would still work if we revert to using `perform_later` within the job rather than using `perform_async` - depending on how Rescue or Delayed Job is set up they might have to change the `perform_later` - but perform_later I think would still be ok without being updated if Sidekiq is configured as the ActiveJob queue adapter